src/Controller/Services.php

```diff
@@ -21,9 +21,15 @@ class Services extends ControllerBase {
   /**
    * Processing the API request.
    */
-  public function processRequest(Request $request, RouteMatchInterface $route_match) {
+
+  public function processRequest(Request $request, RouteMatchInterface $route_match, $service_endpoint_id, $service_definition_id) {
     /** @var $service_endpoint \Drupal\services\ServiceEndpointInterface */
-    $service_endpoint = \Drupal::getContainer()->get('service_manager')->getServiceByEndpoint($request->getPathInfo());
-    return new Response($service_endpoint->processRequest($request));
+    $service_endpoint = \Drupal::entityManager()->getStorage('service_endpoint')->load($service_endpoint_id);
```

Comment:
Please do not remove, this will be used a lot more moving forward, e.g. Delivery Format (selecting a Serialization class to change the response)